website/archiver/tasks.py

```diff
@@ -201,6 +201,8 @@ def archive_addon(addon_short_name, job_pk, stat_result):
     addon_name = addon_short_name
     if 'dataverse' in addon_short_name:
         addon_name = 'dataverse'
+        revision = 'latest' if addon_short_name.split('-')[-1] == 'draft' else 'latest-published'
```

Comment:
Don't change this, because it's done elsewhere in the addon and is out of scope, but getting the published status from the short name would be better as a function than by taking the last item from the split just in case it changes at some point, then it's both explicit and localized to change. I just mention because it gives me something to talk about.